Team6220_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team6220_CENTERSTAGE/MecanumDrive.java

```diff
@@ -182,30 +253,65 @@ public MecanumDrive(HardwareMap hardwareMap, Pose2d pose) {
         leftBack = hardwareMap.get(DcMotorEx.class, "motBL");
         rightBack = hardwareMap.get(DcMotorEx.class, "motBR");
         rightFront = hardwareMap.get(DcMotorEx.class, "motFR");
-        intakeMotor = hardwareMap.get(DcMotorEx.class, "intakeMotor");
+        if (!isDevBot) { // is competition bot
+            intakeMotor = hardwareMap.get(DcMotorEx.class, "intakeMotor");
+            slideMotor = hardwareMap.get(DcMotorEx.class, "motSlides");
+            returnMotor = hardwareMap.get(DcMotorEx.class, "motReturn");
+            motSuspension = hardwareMap.get(DcMotorEx.class, "motSuspension");
+            droneServo = hardwareMap.get(Servo.class, "droneServo");
+            intakeServo = hardwareMap.get(ServoImplEx.class, "intakeServo");
+            dumperServo = hardwareMap.get(ServoImplEx.class, "outtakeServo");
+            pixelLatchFront = hardwareMap.get(ServoImplEx.class, "pixelLatchFront");
+            pixelLatchBack = hardwareMap.get(ServoImplEx.class, "pixelLatchBack");
+        }
 
-        // reverse fr and br motors so that it drives correctly
         rightFront.setDirection(DcMotorEx.Direction.REVERSE);
         rightBack.setDirection(DcMotorEx.Direction.REVERSE);
 
         leftFront.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
         leftBack.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
         rightBack.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
         rightFront.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
-        intakeMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+        if (!isDevBot) { // is competition bot
+            intakeMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+            slideMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+            motSuspension.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
 
-        // now has been enabled, encoders are goodge :D
-        leftFront.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
-        leftBack.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
-        rightBack.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
-        rightFront.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+            returnMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.FLOAT);
+        }
 
-        intakeMotor.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
+        if(!isDevBot) {
+            // preset servo positions
+            intakeServo.setPosition(Constants.INTAKE_POSITIONS[Constants.INTAKE_POSITIONS.length - 1]);
+            dumperServo.setPosition((Constants.DUMPER_INITIALIZATION_POS));
+            pixelLatchFront.setPosition(Constants.PIXEL_LATCH_POSITIONS[0]);
+            pixelLatchBack.setPosition(Constants.PIXEL_LATCH_POSITIONS[0]);
+        }
+
+        // now has been enabled, encoders are goodge :D
```

Comment:
I think this comment ("now has been enabled, encoders are goodge :D") is out-of-date because RUN_USING_ENCODER is now disabled?  (Road Runner still uses encoders even when RUN_WITHOUT_ENCODER is specified for the motors, of course.)